test_cases/en.json

```diff
@@ -260,9 +263,11 @@
       ["starting-in", "very-light-rain", ["less-than", ["minutes", 1]]]],
 
   "Next hour forecasts are temporarily unavailable due to all nearby radar stations being offline.":
-  ["sentence",["next-hour-forecast-status", "temporarily-unavailable", "station-offline"]],
+    ["sentence", ["next-hour-forecast-status", "temporarily-unavailable", "station-offline"]],
```

Comment:
we should make the linter yell about these since otherwise i'll keep doing it 😞

Reply:
The linter doesn't check JSON :(

```
midori:translations jay$ ./node_modules/.bin/eslint test_cases/en.json 

/Users/jay/Projects/Dark Sky/translations/test_cases/en.json
  2:10  error  Parsing error: Unexpected token :

✖ 1 problem (1 error, 0 warnings)

```

Reply:
😭